huaweicloud/resource_huaweicloud_compute_volume_attach.go

```diff
@@ -127,36 +128,27 @@ func resourceComputeVolumeAttachV2Read(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
-	attachment, err := volumeattach.Get(computeClient, instanceId, attachmentId).Extract()
+	attachment, err := block_devices.Get(computeClient, instanceId, attachmentId).Extract()
```

Comment:
suggest to replace attachmentId volumeId
also do it in functions parseComputeVolumeAttachmentId

Reply:
yeah, the `attachmentId` is inaccurate.